### DISCUSSION TITLE
When not following a lot of users, suggestions can be repeated

### DISCUSSION THREAD

**Discussion by HectorCastelli** [community/community#17651] [Other Feature Feedback, Questions, & Ideas] [answered]
I don't follow a lot of GitHub users, so there are not a lot of suggestions that can be given to me (I think I get 6 unique users suggested).

When using the new feed, on every new page, the user follow suggestions ("Recommended based on people you follow") are repeated.

**Comment by siminapasat** [comment]
Thanks for your feedback @HectorCastelli . We'll look into this!

**Comment by JnBrymn** [comment, marked as answer]
Hey there @HectorCastelli, perhaps we misunderstand the problem you're having. Do you mean:

1. You go to the homepage, scroll to the bottom, and click MORE and as more cards load on the page, you see duplicate user recommendations? OR
2. You went to the homepage yesterday and today and always see the same users recommended, but no duplicate cards.

If it's number 1, we're having trouble replicating the problem. Would you consider sending us a screenshot so that we can see what it looks like on your side? If you're concerned about publishing it here, you can send it to [my email](mailto:jnbrymn@github.com). Also, check and see if you're still seeing the same behavior. We made some changes recently that might have affected this behavior.

If it's number 2, then I encourage you to follow and collaborate with more people! We promise that our recommendations will eventually become more interesting 😄

**Reply by HectorCastelli** [reply]
Hello @JnBrymn,

It was the first scenario, but, as you said, it no longer happens.

Memory might be fuzzy but I believe the homepage would have multiple "MORE" clicks available on it, and after a couple of them, the behaviour would trigger.
I just tested it now and noticed I couldn't click "MORE" quite the same number of times, therefore the bug didn't happen.

LGTM ;)


Thanks for the quick replies!

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
